salt/_modules/metalk8s_service_configuration.py

```diff
@@ -35,13 +38,19 @@ def get_service_conf(
 
     .. code-block:: bash
 
-        salt-call metalk8s_service_configuration.get_service_conf "metalk8s-monitoring" "metalk8s-prometheus-config"
+        salt-call metalk8s_service_configuration.get_service_conf "metalk8s-monitoring" "metalk8s-prometheus-config" default_csc
```

Comment:
Your example should be valid (or at least what's `default_csc`), this is not the case here, we don't really understand what's `default_csc`.

Reply:
Done